src/sources/util/http/prelude.rs

```diff
@@ -57,8 +57,8 @@ pub trait HttpSource: Clone + Send + Sync + 'static {
         Ok(Box::pin(async move {
             let span = crate::trace::current_span();
             let mut filter: BoxedFilter<()> = warp::post().boxed();
-            // https://github.com/rust-lang/rust-clippy/issues/8148
-            #[allow(clippy::unnecessary_to_owned)]
+            // // https://github.com/rust-lang/rust-clippy/issues/8148
```

Comment:
Temporarily commented this out to appease clippy, presumably after the download to 1.58.1.